DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -479,7 +481,13 @@ public DMASTStatement Statement(bool requireDelimiter = true)
                         Whitespace();
                         procStatements.Add(statement);
                     } else {
-                        if (procStatements.Count == 0) return null;
+                        if (procStatements.Count == 0)
+                        {
+                            // Sometimes ';' gets used in NOP blocks
+                            Delimiter();
+                            Whitespace();
+                            return null;
+                        }
```

Comment:
This isn't the correct way to go about this, since this code should be valid:
```
switch (1)
    if (1)
        ;
        usr << "One";
```
It would be better to parse every empty statement in the block and not return early if the first one is empty. You can do that by removing this entire else block and moving the original `if (procStatements.Count == 0) return null` line above the `return procStatements` one.